bus-mall.js

```diff
@@ -0,0 +1,109 @@
+// refresh page, .random choseing imgags, images stored in array, <15, create counter/
+//whileloop. event listner < on click refresh page
+
+
+var productNames = ['bag', 'banana', 'bathroom', 'boots', 'breakfast', 'bubblegum', 'chair', 'cthulhu', 'dog-duck', 'dragon', 'pen', 'pet-sweep', 'scissors', 'shark', 'tauntaun', 'unicorn', 'water-can', 'wine-glass'];
+
+var productArr = [];
```

Comment:
Can you think of a clearer name for this? Maybe allProducts? 

Also, as an aside, it's generally considered a no-no to describe the type of a variable in the name of it, like specifying that it's an array. That tells you what something is, but it gives zero information about what it's for. It's ok while you're learning to code, but it's something to look out for in the future.